src/features/emojiMod.ts

```diff
@@ -69,6 +70,59 @@ export const reactionHandlers: ReactionHandlers = {
 
     reportUser({ reason, message, staff, members });
   },
+  "🔍": async ({ message, usersWhoReacted }) => {
+    const KNOWN_REACTOR_THRESHOLD = 1;
+    const UNKNOWN_REACTOR_THRESHOLD = 2;
+
+    const [unknownReactors, knownReactors] = partition(
+      (user) => isStaff(user) || isHelpful(user),
+      usersWhoReacted,
+    );
+
+    if (
+      !(
+        knownReactors.length >= KNOWN_REACTOR_THRESHOLD ||
+        unknownReactors.length >= UNKNOWN_REACTOR_THRESHOLD
+      ) ||
+      message.channel.isThread()
+    ) {
+      return;
+    }
+
+    const newThreadName = `Sorry ${message.author.username}, your question needs some work`;
+
+    const newThread = await message.startThread({
+      name: newThreadName,
+    });
+
+    await newThread.send({
+      embeds: [
+        {
+          title: "Please improve your question",
+          type: "rich",
+          description: `
+Sorry, our most active helpers have flagged this as a question that needs more work before a good answer can be given. This may be because it's ambiguous, too broad, or otherwise challenging to answer. 
+
+This is a good resource about asking good programming questions: 
+
+https://zellwk.com/blog/asking-questions/
+
+(this was triggered by crossing a threshold of "🔍" reactions on the original message)
+          `,
+          color: EMBED_COLOR,
+        },
+      ],
+    });
+
+    await message.delete();
+
+    reportUser({
+      reason: ReportReasons.lowEffortQuestionRemoved,
+      message,
+      staff: unknownReactors,
+      members: knownReactors,
```

Comment:
I think we need to switch these around. Aside from that it looks good to go 🎉 

```suggestion
      staff: knownReactors,
      members: unknownReactors,
```